en/flight_controller/pixhawk.md

```diff
@@ -229,6 +229,18 @@ Pin | Signal | Volt
 2 (blk) | !IO_LED_SAFETY | GND
 3 (blk) | SAFETY   | GND
 
+
+## Serial Port Mapping
+
+UART | Device | Port
+--- | --- | ---
+UART1 | /dev/ttyS0 | IO debug (except v4, there ttyS0 is the wifi)
```

Comment:
So in V4, what would this look like?
```
UART1 | /dev/ttyS0 | Wifi?

```

Reply:
@pkocmoud Would you please update this

Reply:
FYI, I fixed up this one to what the text said for FMUv2. Examples that need fixing for FMUv4 are pixracer (https://github.com/PX4/px4_user_guide/pull/672/files#r388713805) and pixhawk3pro (https://github.com/PX4/px4_user_guide/pull/672/files#r388713805)